specification/appconfiguration/data-plane/Microsoft.AppConfiguration/stable/2023-11-01/appconfiguration.json

```diff
@@ -259,19 +259,19 @@
           {
             "name": "key",
             "in": "query",
-            "description": "A filter used to match keys.",
+            "description": "A filter used to match keys. Syntax reference: https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-key-value#filtering",
             "type": "string"
           },
           {
             "name": "label",
             "in": "query",
-            "description": "A filter used to match labels",
+            "description": "A filter used to match labels. Syntax reference: https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-key-value#filtering",
             "type": "string"
           },
           {
             "name": "tags",
             "in": "query",
-            "description": "A filter used to query by tags",
+            "description": "A filter used to query by tags. Syntax reference: https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-key-value#filtering",
```

Comment:
Microsoft docs links shouldn't have the language. "en-us" here. 

Also, it would be a good idea to have an aka.ms link.

Reply:
Updated, thanks!